src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -280,7 +280,7 @@ public static function createFromGlobals()
 
         $request = self::createRequestFromFactory($_GET, $_POST, array(), $_COOKIE, $_FILES, $server);
 
-        if (0 === strpos($request->headers->get('CONTENT_TYPE'), 'application/x-www-form-urlencoded')
+        if (in_array($request->headers->get('CONTENT_TYPE'), array('application/x-www-form-urlencoded', 'multipart/form-data'))
```

Comment:
how can that be valid? I mean: a body of type multipart/form-data cannot be parsed by `parse_str()`, which is what this `if` block does. Do I miss something?